modules/security-context-constraints-about.adoc

```diff
@@ -124,6 +125,12 @@ The `restricted` SCC:
 * Requires that a pod is run with a pre-allocated MCS label
 * Allows pods to use any FSGroup
 * Allows pods to use any supplemental group
+
+[NOTE]
+====
+The `restricted` SCC still configures `allowPrivilegeEscalation: true`, which disables the `no_new_privs` protection. If you want to prevent this in your cluster, you can create a custom SCC with the flag disabled and configure the access to only appropriate users in place of `restricted`. For more details about flag, see link:https://www.kernel.org/doc/html/latest/userspace-api/no_new_privs.html[The Linux Kernel documentation].
```

Comment:
I get confused from "If you want to prevent this in your cluster". What is "this"? Configuring the first key? Disabling the second? Both? 

It's also not clear exactly what "the flag" refers to. Can you make all of this explicit?

Reply:
@dbaker-rh need your suggestion here.

Reply:
edit: ... which disables the "no_new_privs" protection flag.

edit: ... if you want the additional protection from disabling no_new_privs, you can create ...